src/test/sleep/test_sleep.rs

```diff
@@ -125,65 +111,6 @@ fn sleep_and_signal_test() {
     }
 }
 
-fn clock_nanosleep_with_abstime() {
-    let clock = libc::CLOCK_MONOTONIC;
-
-    // get time time before sleeping
-    let before = call_clock_gettime(clock);
```

Comment:
Do we have just a basic `clock_nanosleep` test with `TIMER_ABSTIME` in the new tests?

Reply:
Yeah, that was merged in #3026. 

https://github.com/shadow/shadow/blob/a13765dd2f58990d3fb4361967a2f67b3c5da4cd/src/test/time/clock_nanosleep/test_clock_nanosleep.rs#L67